src/ranged.cpp

```diff
@@ -103,11 +103,14 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
 {
     const bool do_animation = OPTIONS["ANIMATIONS"];
 
-    double range = rl_dist(source, target_arg);
-    // .013 * trange is a computationally cheap version of finding the tangent in degrees.
-    // 0.0002166... is used because the unit of dispersion is MOA (1/60 degree).
-    // It's also generous; missed_by will be rather short.
-    double missed_by = shot_dispersion * 0.00021666666666666666 * range;
+    constexpr double dblpi = 6.28318530717958f;
```

Comment:
Why not `M_PI * 2`? And why are you initializing a `double` with a `float` (the `f` at the end of the literal)?